### DISCUSSION TITLE
smplayer from snap - opening file from long path not working

### DISCUSSION THREAD

**Discussion by andreascschmidt** [smplayer-dev/smplayer#372] [Q&A]
I have Ubuntu 22.04 and used smplayer from snap 
smplayer                           21.10.0               43     latest/stable    rvm             -

When I try to open a file from a very long path, I'm unable to do that.

`/snap/smplayer/43/usr/bin/mpv --no-config --no-quiet --terminal --no-msg-color --input-ipc-server=/tmp/smplayer-mpv-2cb4d --msg-level=ffmpeg/demuxer=error --video-rotate=no --no-fs --hwdec=no --sub-auto=fuzzy --shm-buffer-name=/smplayer-183117 --vo=shm, --no-input-default-bindings --input-vo-keyboard=no --no-input-cursor --cursor-autohide=no --no-keepaspect --monitorpixelaspect=1 --osd-level=1 --osd-scale=1 --osd-bar-align-y=0.6 --sub-ass --embeddedfonts --sub-ass-line-spacing=0 --sub-scale=1 --sub-font=Arial --sub-color=#ffffffff --sub-shadow-color=#ff000000 --sub-border-color=#ff000000 --sub-border-size=0.75 --sub-shadow-offset=2.5 --sub-font-size=50 --sub-bold=no --sub-italic=no --sub-margin-y=8 --sub-margin-x=20 --sub-codepage=ISO-8859-1 --sub-pos=100 --volume=110 --cache=auto --screenshot-template=cap_%F_%p_%02n --screenshot-format=jpg --screenshot-directory=/home/andreas/Pictures/smplayer_screenshots --audio-pitch-correction=yes --volume-max=110 --term-playing-msg=MPV_VERSION=${=mpv-version:}
INFO_VIDEO_WIDTH=${=width}
INFO_VIDEO_HEIGHT=${=height}
INFO_VIDEO_ASPECT=${=video-params/aspect}
INFO_VIDEO_FPS=${=container-fps:${=fps}}
INFO_VIDEO_FORMAT=${=video-format}
INFO_VIDEO_CODEC=${=video-codec}
INFO_DEMUX_ROTATION=${=track-list/0/demux-rotation}
INFO_AUDIO_FORMAT=${=audio-codec-name}
INFO_AUDIO_CODEC=${=audio-codec}
INFO_AUDIO_RATE=${=audio-params/samplerate}
INFO_AUDIO_NCH=${=audio-params/channel-count}
INFO_LENGTH=${=duration:${=length}}
INFO_DEMUXER=${=current-demuxer:${=demuxer}}
INFO_SEEKABLE=${=seekable}
INFO_TITLES=${=disc-titles}
INFO_CHAPTERS=${=chapters}
INFO_TRACKS_COUNT=${=track-list/count}
METADATA_TITLE=${metadata/by-key/title:}
METADATA_ARTIST=${metadata/by-key/artist:}
METADATA_ALBUM=${metadata/by-key/album:}
METADATA_GENRE=${metadata/by-key/genre:}
METADATA_DATE=${metadata/by-key/date:}
METADATA_TRACK=${metadata/by-key/track:}
METADATA_COPYRIGHT=${metadata/by-key/copyright:}
INFO_MEDIA_TITLE=${=media-title:}
INFO_STREAM_PATH=${stream-path}
 --audio-client-name=SMPlayer --term-status-msg=STATUS: ${=time-pos} / ${=duration:${=length:0}} P: ${=pause} B: ${=paused-for-cache} I: ${=core-idle} VB: ${=video-bitrate:0} AB: ${=audio-bitrate:0} /home/andreas/Downloads/net/Dropbox/Z MP3 Store/Z Music Videos/Klassic - Orchestra - Konzerte/The Best of Paganini.mp4

[file] Cannot open file '/home/andreas/Downloads/net/Dropbox/Z MP3 Store/Z Music Videos/Klassic - Orchestra - Konzerte/The Best of Paganini.mp4': Permission denied
Failed to open /home/andreas/Downloads/net/Dropbox/Z MP3 Store/Z Music Videos/Klassic - Orchestra - Konzerte/The Best of Paganini.mp4.
Exiting... (Errors when loading file)
`

It says permission denied, but my direct installed mvp player can play that well. 
I noted using the smplayer from the Ubuntu repo which was outdated, and  only recently updated to 
smplayer/jammy 21.10.0~ds0-1 amd64 can play that file well too. I'm not sure if this is not a snap problem, not being able to handle long path names.

**Comment by smplayer-dev** [comment]
I can't reproduce this problem in my computer.
Is that folder in another drive?

**Comment by andreascschmidt** [comment]
Yes this is a NFS share from a NAS. Permissions are fine.
I went back to the ppa version (ditched the snap) and it works there again now. 
It looks not to be related to SMplayer, but potentially with snap. I spotted the same issue with VLC from snap.
